packages/analytics/dashboard-renderer/src/types/dashboard-renderer-types.ts

```diff
@@ -390,10 +390,17 @@ export type TileLayout = FromSchema<typeof tileLayoutSchema>
 export const tileConfigSchema = {
   type: 'object',
   properties: {
+    id: {
```

Comment:
I kind of feel like the slottable tile should be a new "chart type" with `id` as a property -- otherwise, you could potentially have a tile that's slottable AND has a chart, and I feel like that doesn't make sense.  Happy to talk about it, though.

Reply:
I started going down that route, but it too messy for my liking and forced the user of the dashboard to include redundant properties. There's no need for a "chart" and there's no need for a "query" for a slottable tile. 

As a result, a slottable component does not need anything from "definition" at all. I'd rather simplify and cut out the definition altogether for a slottable tile than over complicate further up the chain, just to avoid the one scenario you mention which can be cleared up in the documentation or with a console warn. 

Also, it forces us to add a `slottable` enum to the ChartTypes enum to satisfy the renderer lookup. That's also unneeded since with a slottable tile, the `DashboardTile` component doesn't even get rendered, and the code never even gets there with a slottable tile. 

Let me know if there's any other situations where we might want to pass additional information and context to a slottable tile that might require it to be a new "chart type"
(which I don't think is the case since the host app will be declaring it in its template)

Reply:
Passing through the `syntheticsDataKey` to the slot might be a good idea

Reply:
Ok, made a change to try and get the best of both worlds.

Keep the tile definition consistent while avoiding doing the nested slots 😬